libs/io_repl.c

```diff
@@ -95,8 +97,13 @@ void b_putchar(int idx, int *cells) {
 }
 
 void b_float_print(int idx, int *cells) {
-  float value = cells[idx] / 100.0;
-  printf("%.2f", value);
+//  float value = cells[idx] / 100.0;
+//  printf("%.2f", value);
+printf(
```

Comment:
Isn't a mistake over here? Probably it should be:

```
float value = cells[idx]/100.0; 
print("Index Pointer: %d Value at Index Pointer: %.2f\n", idx, value);
```
Since this is the float print function, right?